### DISCUSSION TITLE
Fly no longer offers a free tier

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#816] [Q&A]
Fly used to have a free hobby plan:
- No monthly subscription
- No included usage with that subscription, obviously
- Free allowances

Then they increased that to the paid hobby plan:
- $5/mo subscription
- $5 of included usage
- Free resource allowances

Then they increased that to the Pay As You Go plan:
- No monthly subscription
- No free resource allowances
- Pay for what you use

So, at the moment, you have:
- Pay As You Go plan
- $29/mo Launch plan
- $199/mo Scale plan

Unfortunately I don't know of any other platform that supports quickstart VMs and Volumes with native LiteFS support for easy scalability with SQLite. Fly seems to be relatively unique in that regard.

And even though it is possible to configure LiteFS manually in these platforms:
- Heroku
- Render
- DigitalOcean
- Railway
They cost more than Fly.io.

I wonder if anyone knows of a platform that offers a generous free tier.

**Comment by kentcdodds** [comment]
Thanks for the context. I'm pretty confident that even with these changes you can deploy the Epic Stack and have everything live within their free allowances as long as you don't need more storage or a bunch of instances.

Being free is not a primary goal of the Epic Stack, but it's nice to have if it's possible.

I received an email recently telling me that LiteFS Cloud (where data backups are stored and disaster recovery is possible) will be deprecated and removed in October, so I'm in talks with Fly, Turso, and Prisma looking for an alternative solution for this (because data backups and disaster recovery is important).

I do not plan on leaving Fly.io as the host for Epic Stack apps, but it's possible we'll move from using volumes and LiteFS. Stay tuned.

**Reply by andrecasal** [reply]
No, there are no free allowances on the Pay As You Go plan.

Free allowances are only included in:
- Free and Paid Hobby plan (deprecated)
- $29/mo Launch plan
- $199/mo Scale plan
- Entreprise plan

Free allowances (for reference):
- 3x shared-cpu-1x 256MB VMs
- 3GB total persistent volume storage
- Outbound data transfer:
  - 100 GB North America & Europe
  - 30 GB Asia Pacific, Oceania & South America
  - 30 GB Africa & India

**Reply by kentcdodds** [reply]
They also have a policy of forgiveness of charges of less than $5 which applies to many hobby projects. This is undocumented and could go away at any time, but I get an email about it every month for simple apps I have.

**Reply by andrecasal** [reply]
Oh, I thought that _was_ the free allowance. Good to know!

**Reply by stephen776** [reply]
> Thanks for the context. I'm pretty confident that even with these changes you can deploy the Epic Stack and have everything live within their free allowances as long as you don't need more storage or a bunch of instances.
> 
> 
> 
> Being free is not a primary goal of the Epic Stack, but it's nice to have if it's possible.
> 
> 
> 
> I received an email recently telling me that LiteFS Cloud (where data backups are stored and disaster recovery is possible) will be deprecated and removed in October, so I'm in talks with Fly, Turso, and Prisma looking for an alternative solution for this (because data backups and disaster recovery is important).
> 
> 
> 
> I do not plan on leaving Fly.io as the host for Epic Stack apps, but it's possible we'll move from using volumes and LiteFS. Stay tuned.
> 
> 
> 
> 

Any updates on the LiteFS Cloud alternative?

**Reply by kentcdodds** [reply]
None yet. It's being worked on from multiple angles right now. Not sure when anything will be ready to preview. Sorry!
